src/field_10x26.h

```diff
@@ -30,7 +30,10 @@ typedef struct {
      *     sum(i=0..9, n[i] << (i*26)) < p
      *     (together these imply n[9] <= 2^22 - 1)
      */
-    SECP256K1_FE_VERIFY_FIELDS
+#ifdef VERIFY
+    int magnitude;
+    int normalized;
+#endif
 } secp256k1_fe;
```

Comment:
Have you tried/considered including `field.h` instead?

Reply:
Doesn't it create a circular dependency?

Reply:
In some sense, yes, but it's not a problem because the include guards should break any loops.

If you take that to the extreme, we could also have a single `headers.h` that includes *all* internal headers and include `headers.h` (almost) everywhere. That's a rather simple solution. I'm not convinced that this is very elegant, but it gets the job done...

Reply:
> it's not a problem because the include guards should break any loops.

Right. But not for renaming a header to a source file. It is not clear (at least for me) how to make it meaningful. FWIW, I've already tried the testing script that keeps an original header unmodified.

Reply:
> Right. But not for renaming a header to a source file. It is not clear (at least for me) how to make it meaningful.

You mean problems will occur if we move code from a header to a source file?

> FWIW, I've already tried the testing script that keeps an original header unmodified.

I don't understand. Can you rephrase?

Reply:
> > Right. But not for renaming a header to a source file. It is not clear (at least for me) how to make it meaningful.
> 
> You mean problems will occur if we move code from a header to a source file?

Yes. I mean:
```sh
mv src/field_10x26.h src/field_10x26.c
gcc -c src/field_10x26.c -o src/field_10x26.o
```

> > FWIW, I've already tried the testing script that keeps an original header unmodified.
> 
> I don't understand. Can you rephrase?

A modified script looks like that:
```sh
cp src/field_10x26.h src/field_10x26.c
gcc -c src/field_10x26.c -o src/field_10x26.o
```